packages/ember-testing/tests/helpers_test.js

```diff
@@ -846,11 +841,9 @@ if (!jQueryDisabled) {
 
         this.addTemplate(
           'index',
-          `
-        <input type="text" id="first"
```

Comment:
This one was using the `<input>` HTML element before, so we shouldn't change it. I didn't actually check if it was _intentional_ or _important_ to the test that it uses `<input>` instead of `<Input />`, but we shouldn't _need_ to change that as part of this PR so better to leave it as-is. If there is a good reason to update those tests separately from landing this deprecation, we could always do that as a separate PR to make it clear.